### DISCUSSION TITLE
Can't accept invitation

### DISCUSSION THREAD

**Discussion by shota-akizuki-eda** [community/community#29187] [Other Feature Feedback, Questions, & Ideas] [answered]
Dear all
I received an invitation to join a private repository, but the invitation page is scrolling  infinitely.
I can't press the accept button.
It's like [this](https://twitter.com/IllesAprod/status/1557352505088016386?s=20&t=PTszMktWU1x0aMMGm2DQ6A).
Can I fix that myself ?

**Comment by verduscos** [comment]
I've seen this issue with another user recently. Have you tried using a different browser?

**Reply by shota-akizuki-eda** [reply]
Thank you for replying.
I tried using Chrome, Firefox, Safari.
In addition to them, I tried with My iPhone（Safari / Chrome / Firefox）, but the page is scrolling infinitely.

**Reply by shota-akizuki-eda** [reply]
There seems to be no problem on your invitation.（Probably. I can press the button.）

https://user-images.githubusercontent.com/110839014/184220535-85e18891-8728-41eb-a5d5-1ae08edceccf.mov

**Reply by shota-akizuki-eda** [reply]
Two days ago My client sent two invitations. And the both are until the same situiation.

https://user-images.githubusercontent.com/110839014/184221668-1006eec1-7bef-42f5-8a2d-b98fe2b14bce.mov

**Reply by verduscos** [reply]
If you are able to join with my invite, then I suspect that it is an issue with that repo, I'd ask for a new invite link. I am not sure what the issue may actually be though.

**Comment by radar-blip** [comment, marked as answer]
Similar situation here: #29167

Where to file a bug report here: https://support.github.com/contact/bug-report

**Comment by shota-akizuki-eda** [comment]
Many thanks to you all.
Just now, I was able to accept the invitation but I don't know why😅
It could be fixed by GitHub.
